c2cgeoportal/tests/functional/test_entry.py

```diff
@@ -51,6 +52,7 @@
 
 @attr(functional=True)
 class TestEntryView(TestCase):
+    _tables = None
```

Comment:
as in test_themes_edit_columns.py  the table should be in the setUp :-) init as an empty array